tests/fixtures/generate_test_pdfs.py

```
@@ -0,0 +1,228 @@
+#!/usr/bin/env python3
+"""Generate test PDFs for weave-cli PDF extraction tests (Issue #8)."""
+
+import os
+import random
+from fpdf import FPDF
+from PIL import Image
+
+BASE = "/tmp/weave-cli/tests/fixtures"
```

Ruff (0.15.7)

[error] Probable insecure usage of temporary file or directory: "/tmp/weave-cli/tests/fixtures"

(S108)